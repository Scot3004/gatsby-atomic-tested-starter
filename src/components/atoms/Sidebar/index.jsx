import styled from 'styled-components';

export const Sidebar = styled.nav`
  grid-area: sidebar;
  min-width: 0;
`;

export default Sidebar;
