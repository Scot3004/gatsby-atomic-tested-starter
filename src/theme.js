const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
