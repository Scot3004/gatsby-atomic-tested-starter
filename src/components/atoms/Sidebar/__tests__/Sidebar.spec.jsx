import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import theme from '../../../../theme';
import Sidebar from '..';

describe('When I visit the site in a desktop', () => {
  const tree = renderer.create(<Sidebar theme={theme} />).toJSON();
  test('Then the profile of the blog owner should be shown in a sidebar', () => {
    expect(tree).toMatchSnapshot();
  });
});
