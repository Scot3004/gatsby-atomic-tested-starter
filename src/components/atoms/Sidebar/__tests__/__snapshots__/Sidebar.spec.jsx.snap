// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I visit the site in a desktop Then the profile of the blog owner should be shown in a sidebar 1`] = `
.c0 {
  grid-area: sidebar;
  min-width: 0;
}

<nav
  className="c0"
/>
`;
